spec/support/helpers.rb

```diff
@@ -17,7 +17,7 @@ def self.bundler_version
 
       versions = bundler_versions
 
-      to_find = ENV["TEST_BUNDLER_VERSION"] || ENV["BUNDLER_VERSION"]
+      to_find = ENV.fetch("TEST_BUNDLER_VERSION", nil) || ENV.fetch("BUNDLER_VERSION", nil)
```

Comment:
This is fine...you could also do:

```suggestion
      to_find = ENV.fetch("TEST_BUNDLER_VERSION", ENV.fetch("BUNDLER_VERSION", nil))
```